### PR TITLE
Add content api tests for connect relation format

### PR DESCRIPTION
**Pull request by petersg83**
### What does it do?

Tests the new relational fields edit format.
- [x] test the content api
- [x] test the admin api
 
### Why is it needed?

To ensure there are no regressions

### How to test it?

Checks if it's green :)

### Related issue(s)/PR(s)

None

### CODE REVIEW

File: packages/core/strapi/tests/api/relations.test.e2e.js [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+              products_mo: { data: { id: id1 } },
+              products_mw: { data: [{ id: id1 }, { id: id2 }] },
+              products_om: { data: [{ id: id1 }, { id: id2 }] },
+              products_oo: { data: { id: id1 } },
+              products_ow: { data: { id: id1 } },
+            },
+          });
+        });
+
+        test('In reversed order', async () => {
```
Comment (Marc-Roig) [inline packages/core/strapi/tests/api/relations.test.e2e.js:209]: curious, why do you check inserting the ids in the reverse order?

File: packages/core/strapi/tests/api/relations.test.e2e.js [hunk 2]
```diff
@@ -0,0 +1,690 @@
...
+    describe.each([
+      ['directly in the array ([3])', 'object'],
+      ['an object in the array ([{ id: 3 }])', 'array'],
+    ])('ids being %s', (name, mode) => {
+      test('Adding id3', async () => {
+        const createdShop = await createEntry(
+          'shops',
+          {
+            name: 'Cazotte Shop',
+            products_ow: { connect: [id1] },
```
Comment (Marc-Roig) [inline packages/core/strapi/tests/api/relations.test.e2e.js:261]: We only have documented the array format [{id: 3}], should we document this other one too?

**Review by Marc-Roig — APPROVED** [review]

File: packages/core/strapi/tests/api/relations.test.e2e.js (hunk 1 above)
Comment (petersg83) [reply packages/core/strapi/tests/api/relations.test.e2e.js:209]: To ensure that having the right order in the test above was not by chance :p

File: packages/core/strapi/tests/api/relations.test.e2e.js (hunk 2 above)
Comment (Marc-Roig) [reply packages/core/strapi/tests/api/relations.test.e2e.js:261]: yes! I think it is a clean way to connect when you do not care about order.

**Review by Marc-Roig — APPROVED** [review]

**Pull request merged**